From 569289211978242171e5671b75457a0f277528eb Mon Sep 17 00:00:00 2001
From: Juhapekka Piiroinen <juhapekka.piiroinen@csc.fi>
Date: Tue, 21 Jan 2020 14:42:16 +0000
Subject: [PATCH] Update development uris.

---
 .env.development | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/.env.development b/.env.development
index d7fc83e..d792200 100644
--- a/.env.development
+++ b/.env.development
@@ -2,12 +2,12 @@
 VUE_APP_ENVIRONMENT="development"
 # VUE_APP_DEV_TOKEN is used for local development login.
 VUE_APP_DEV_TOKEN=""
-VUE_APP_METAX_API_URL=https://metax-test.csc.fi/rest
+VUE_APP_METAX_API_URL=https://metax.csc.local/rest
 VUE_APP_METAX_FILEAPI_URL=/api/proxy
-VUE_APP_ES_API_URL=https://metax-test.csc.fi/es
-VUE_APP_ETSIN_API_URL=https://etsin-test.fairdata.fi/dataset
+VUE_APP_ES_API_URL=https://metax.csc.local/es
+VUE_APP_ETSIN_API_URL=https://etsin.csc.local/dataset
 VUE_APP_QVAIN_API_URL=""
-VUE_APP_MATOMO_SITE_ID="7"
+VUE_APP_MATOMO_SITE_ID="2"
 VUE_APP_MATOMO_SRC="matomo.js"
-VUE_APP_MATOMO_URL="//matomo-test.fairdata.fi/"
+VUE_APP_MATOMO_URL="//matomo.csc.local/"
 VUE_APP_MATOMO_TRACKER_PHP="matomo.php"
-- 
2.16.6

